Update scripting_first_script.rst (#7023)
godotengine/godot-docs: getting_started/step_by_step/scripting_first_script.rst

Type "Sprite2D" in the search bar to filter nodes and double-click on Sprite2D
to create the node.

.. note::

>     Image below shows "Sprite" but in Godot 4 it will be shown as "Sprite2D".

Reviewer: ```suggestion
    The image below shows "Sprite" but in Godot 4 it's "Sprite2D".
```
